src/components/tabs/buy-tab-content/buy.overview.tsx

```diff
@@ -0,0 +1,30 @@
+import { useAssetContext } from '../../../api/contexts/asset.context';
+import { Asset } from '../../../api/definitions/asset';
+import DfxIcon, { IconVariant } from '../../../stories/DfxIcon';
+
+interface BuyTabContentOverviewProps {
+  onAssetClicked: (asset: Asset) => void;
+}
+
+export function BuyTabContentOverview({ onAssetClicked }: BuyTabContentOverviewProps): JSX.Element {
+  const { buyableAssets } = useAssetContext();
+  return (
+    <div className="flex flex-col gap-8">
+      {Array.from(buyableAssets.keys()).map((blockchain, blockchainIndex) => (
```

Comment:
If you use
`Array.from(buyableAssets.entries()).map(([blockchain, assets], blockchainIndex) => (`
you don't need the `buyableAssets.get` at line 19.

Reply:
that is true. but I will need another iterator for assetIndex to be able to show multiple